docs/devs/e2e-tests.md

```diff
@@ -64,7 +64,7 @@ Additionally, the `.env` file contains a range of configuration variables that s
 - Run tests in a specific file. If you want to focus on a particular test file, specify the file name, and only tests within that file will be executed. For example:
 
 ```bash
- npx playwright test Odoo
+ npx playwright test testOdooIntegration
```

Comment:
So let's wait a little but because of [OZ-544](https://mekomsolutions.atlassian.net/browse/OZ-544?focusedCommentId=43617) this will probably eventually become
> `npx playwright test odoo-openmrs-flows`